plugins/kubernetes-backend/src/types/types.ts

```diff
@@ -193,6 +193,8 @@ export interface ClusterDetails {
    * @see dashboardApp
    */
   dashboardParameters?: JsonObject;
+
+  customResources?: CustomResourceMatcher[];
```

Comment:
Like we just discussed - if you add a nice structured comment here then it stands a chance of showing up nicely in the [API reference](https://backstage.io/docs/reference/plugin-kubernetes-backend) if that starts getting generated properly for this package.